projects/ion/src/lib/chip-group/chip-group.component.scss

```diff
@@ -0,0 +1,4 @@
+.chip-group {
+  display: flex;
+  gap: 6px;
```

Comment:
In figma it asks for a gap of 8px, the padding of the items which is 6 12
![image](https://user-images.githubusercontent.com/54484070/229881160-e76b51f0-4d03-4af2-8fee-f43982042793.png)

Reply:
this 6px gap refers to the distance between the chips, it does not change any style of the chip😊